Controls/Mail/Mailman.c

```diff
@@ -11,8 +13,21 @@
  * Once the letter is deposited, signal the semaphore (effectively putting up the flag).
  * Print "Finished!" at the end.
  */
+//extern char* mailbox;
 void depositLetter(void) {
 	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+	//int i;
+	//char max=random()%255;//different lengths at diff times
+	for(int i=0;i<255;i++){
```

Comment:
I know that this was built into the starter project (my bad!) but typically we don't like magic numbers in our code (i.e. random numbers which people don't understand upon first glance). It might be nice to put 256 in a macro in _Mail.h_, and use that wherever you need to. ex:
`
#define MAILBOX_SIZE 256
`

Additionally, this will loop from numbers 0 through 254, as when it gets to 255, **i** will not be less than 255 anymore. This isn't an issue, as you need to put the null terminator at the last char, but I just wanted to make sure you knew.